### PR TITLE
Fix log and spool directory permissions on startup

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

**Comment by matyasselmeci** [comment]
(didn't test this yet)

File: etc/osg/image-config.d/25-setup-condor-submit.sh [hunk 1]
```diff
@@ -51,3 +51,9 @@ for attr in COLLECTOR_MAX_FILE_DESCRIPTORS \
 done
 [[ -s /etc/condor/config.d/01-fdfix.conf ]] && \
     echo "# This file was created by $prog" >> /etc/condor/config.d/01-fdfix.conf
+
+log=$(condor_config_val LOG); ret=$?
+[[ -n $log && $ret == 0 ]] || fail "Couldn't get log directory (value of the LOG config option)"
+spool=$(condor_config_val SPOOL); ret=$?
+[[ -n $spool && $ret == 0 ]] || fail "Couldn't get spool directory (value of the SPOOL config option)"
```
Comment (brianhlin) [inline etc/osg/image-config.d/25-setup-condor-submit.sh]: I appreciate the thoroughness but this feels antithetical to containers. Instead, we should assume the defaults from upstream and tell users to mount their volumes to said defaults.

**Review by brianhlin — CHANGES_REQUESTED** [review]
I think we should simplify this to just the `chown`

**Review by brianhlin — APPROVED** [review]
LGTM!

**Pull request merged**